function-app/adb-to-purview/src/Function.Domain/Helpers/PurviewCustomType.cs

```diff
@@ -765,8 +793,12 @@ public async Task<List<QueryValeuModel>> Query_entities(object filter)
         public async Task<JObject> GetByGuid(string guid)
         {
             var token = this.GetToken();
-            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? "purview-to-adb-demo-openlineage-purview"}.purview.azure.com/catalog/api";
-            var purviewSearchEndpoint = $"{baseurl}/atlas/v2/entity/guid/";
+
+            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.{Environment.GetEnvironmentVariable("ResourceUri") ?? "purview.azure.com"}";
```

Comment:
This breaks anyone's existing appsettings since the current default is https://purview.azure.com. Should this be a constant variable instead of the string being repeated?